pkg/controller/constraint/stats_reporter_test.go

```diff
@@ -0,0 +1,41 @@
+package constraint
+
+import (
+	"testing"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/util"
+	"go.opencensus.io/stats/view"
+)
+
+func TestReportConstraints(t *testing.T) {
+	var expectedValue int64 = 10
```

Comment:
we could also test this with a constraints cache call to `reportTotalConstraints`